### PR TITLE
Update imaging WFM notebook to work with new ess

### PR DESCRIPTION
**Pull request by nvaytet**
Depends on https://github.com/scipp/ess/pull/44

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -16,60 +16,66 @@
...
+
+#     html_context = {'css_files': ['_static/theme_overrides.css']}
+# else:
+#     html_context = {
+#         'css_files': [
+#             '//media.readthedocs.org/css/sphinx_rtd_theme.css',
+#             '//media.readthedocs.org/css/readthedocs-doc-embed.css',
+#             '_static/theme_overrides.css'
+#         ]
+#     }
```
Comment (SimonHeybrock) [inline conf.py:61]: commented code?

File: .azure-pipelines/pull_request.yml [hunk 2]
```diff
@@ -14,10 +14,10 @@ variables:
...
 
 stages:
-  - stage: 'notebook_tests'
-    displayName: 'notebook tests'
-    jobs:
-      - template: test_notebooks.yml
+  # - stage: 'notebook_tests'
+  #   displayName: 'notebook tests'
+  #   jobs:
+  #     - template: test_notebooks.yml
```
Comment (SimonHeybrock) [inline .azure-pipelines/pull_request.yml:20]: disabled?

File: imaging/bragg-edge-imaging-2D.ipynb [hunk 3]
```diff
@@ -85,24 +73,24 @@
...
-    "        pos2d = sc.fold(pos, dim='spectrum', dims=['y', 'x'], shape=[x, x])\n",
-    "        geometry[\"position\"] = pos2d\n",
-    "        if rot is not None:\n",
-    "            geometry[\"rotation\"] = rot\n",
-    "        if shp is not None:\n",
-    "            geometry[\"shape\"] = shp\n",
-    "        geometry[\"x\"] = pos2d.fields.x['y', 0]\n",
-    "        geometry[\"y\"] = pos2d.fields.y['x', 0]\n",
-    "        return geometry"
+    "    from mantid.simpleapi import LoadEmptyInstrument\n",
```
Comment (SimonHeybrock) [inline imaging/bragg-edge-imaging-2D.ipynb:76]: Can you use https://scipp.github.io/scippneutron/generated/scippneutron.load.html#scippneutron.load with the `mantid_alg` parameter? Could that avoid some of the manual steps here?

File: conf.py (hunk 1 above)
Comment (nvaytet) [reply conf.py:61]: yes, this is being deleted in the next PR anyway: https://github.com/scipp/ess-notebooks/pull/42

File: .azure-pipelines/pull_request.yml (hunk 2 above)
Comment (nvaytet) [reply .azure-pipelines/pull_request.yml:20]: yes, this is being deleted in the next PR anyway: https://github.com/scipp/ess-notebooks/pull/42

Comment (nvaytet) [reply .azure-pipelines/pull_request.yml:20]: I was causing seg faults on the builds

**Review by OwenArnold — APPROVED** [review]

**Pull request merged**